### PR TITLE
fix: ignoring src/scenic/simulators in codecov report

### PR DESCRIPTION
**Pull request by abanuelo**
### Description
PR #223 merge is blocked by decreased code coverage directly resulting from adding code to carla simulator.py.

### Issue Link
None, adhoc fix needed to merge in PR #223 

### Checklist
- [X] I have tested the changes locally via `pytest` and/or other means
- [ ] I have added or updated relevant documentation
- [ ] I have autoformatted the code with black and isort
- [ ] I have added test cases (if applicable)

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -15,6 +15,8 @@ codecov:
     layout: "reach, diff, flags, files"
     behavior: default
   require_ci_to_pass: true
```
Comment (abanuelo) [inline codecov.yml:17]: nit: we can also remove this to ensure it never blocks CI, let me know your thoughts here!

Comment (abanuelo) [reply codecov.yml:17]: Resolving conversation as we have agreed to keep it!

File: codecov.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ codecov:
...
+  ignore:
+    - "src/scenic/simulators/**"
```
Comment (dfremont) [inline codecov.yml:19]: Is there a clean way to include `simulators/newtonian` and `simulators/utils` without explicitly listing all the other subfolders?

Comment (abanuelo) [reply codecov.yml:19]: Yes I can apply a `!` operator! I am going to make that change and let it percolate to see if it has any incompatibilities with `codecov.yml` formatting!

**Review by dfremont — APPROVED** [review]
Looks great!

**Pull request merged**